app/policies/idv/flow_policy.rb

```diff
@@ -18,6 +18,13 @@ def info_for_latest_step
       steps[latest_step]
     end
 
+    def undo_steps_from_controller!(controller:)
+      controller_name = controller.ancestors.include?(ApplicationController) ?
+                        controller.controller_name : controller
```

Comment:
is `controller` a Class or an instance?

If it's a `Class` we can do `<` to check subclass instead of iterating ancestors
```suggestion
      controller_name = controller < ApplicationController ?
                        controller.controller_name : controller
```

Reply:
That works! Cool syntax, if a bit opaque. Updated.